dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -144,6 +260,15 @@
     return externalMethodrefs;
   }
 
+  static List<FullyQualifiedMethodSignature> listMethodsOnClass(JavaClass javaClass) {
+    List<MethodSignature> methods = listDeclaredMethods(javaClass);
+    List<FullyQualifiedMethodSignature> fullyQualifiedMethodSignatures = methods.stream()
```

Comment:
For a different PR, why doesn't `FQMS` have a class name and a `MethodSignature`?